Resources/Public/Css/recordlist.css

```
@@ -13,12 +13,20 @@
     gap: 3px;
 }
 
+.text-inline-edit .inline-edit:hover {
+    outline: Highlight auto 1px;
+    outline: -webkit-focus-ring-color auto 1px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

outline is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)